apps/service_define/config/dev.exs

```diff
@@ -1,5 +1,20 @@
 use Mix.Config
 
+config :service_define, Define.Application,
+  kafka_endpoints: nil,
+  brook: [
+    driver: [
+      module: Brook.Driver.Default,
+      init_arg: []
+    ],
+    handlers: [Define.Event.Handler],
+    storage: [
+      module: Brook.Storage.Ets,
+      init_arg: []
+    ],
+    dispatcher: Brook.Dispatcher.Noop
```

Comment:
Since we're supplying these values "at runtime" in the tests by doing overrides using the `Properties` macro, are we including these here for running the application locally while doing development of the UI? if so, you should be able to remove the same settings in `test.exs` and supply the values in your specific tests by the `temp_env` module in the `:testing` umbrella app.

Reply:
I don't really know anything about the testing umbrella app. Why would we put our configuration there instead of with the apps?

Reply:
Testing gives you a module called `Temp.Env` that lets you set app env configs in a specific test to avoid blanket configurations across the test environment and keeps the `test.exs` and `integration.exs` files much cleaner. It takes the app env stuff that we're setting via the Properties library (also umbrella) and configuring them at runtime, both defaulting to prod settings and overriding only as needed in the tests that require it.

Look at the service_persist's `lib/event/handler.ex` and its associated test to see how we're defining configuration and then overriding it as needed in a test.

Reply:
It seems simpler to just to provide them through the config for now until we have a reason to override them in a test.

Reply:
So that we can run the app locally without any dependencies. I guess we could add it to the top level umbrella base config but that seems weird to me.